ansible/roles/operator-pipeline/tasks/operator-pipeline-import-index-images.yml

```diff
@@ -39,7 +39,7 @@
 - name: Import redhat-marketplace-index imagestream
   tags:
     - import-index-images
-  no_log: true
+  #no_log: true
```

Comment:
Did you forget to uncomment this before committing?

Reply:
I did, thanks for pointing it out.. ti should be fixed now